ui/app/styles/core/charts.scss

```diff
@@ -337,7 +328,7 @@ p.data-details {
   }
 
   .legend-center {
-    grid-column-start: 1;
+    grid-column-start: 2;
```

Comment:
This change was necessary for the dual chart view so the legend doesn't cut off sync clients
<img width="1089" alt="Screenshot 2024-01-17 at 11 51 11 AM" src="https://github.com/hashicorp/vault/assets/68122737/912b2393-e271-440e-87bc-19786e34bf66">